apluslms_roman/utils/path_mapping.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import PurePosixPath
+import json
+import logging
+import re
+from os import environ
+
+
+logger = logging.getLogger(__name__)
+json_re = re.compile(r'^(?:["[{]|(?:-?[1-9]\d*(?:\.\d+)?|null|true|false)$)')
+
+
+def get_host_path(original, mapping):
+    ret = original
+    orig_path = PurePosixPath(original)
+    for k, v in mapping.items():
```

Comment:
Use longer variable names than just k and v. They are some times ok in comprehensions, but this loop isn't very clear about their meaning,